pkg/image/admission/admission_test.go

```diff
@@ -173,7 +174,7 @@ func TestLimitAppliestoImages(t *testing.T) {
 		},
 	}
 
-	plugin, err := NewImageLimitRangerPlugin(fake.NewSimpleClientset(), nil)
+	plugin, err := NewImageLimitRangerPlugin(nil)
```

Comment:
Not needed since this test stress tests `SupportsLimit` method and that doesn't rely on the client, so we're good.